lib/matplotlib/tests/test_patches.py

```diff
@@ -5,18 +5,23 @@
 from numpy.testing import assert_almost_equal, assert_array_equal
 import pytest
 
-from matplotlib.patches import Polygon, Rectangle, FancyArrowPatch
+from matplotlib.patches import Patch, Polygon, Rectangle, FancyArrowPatch
 from matplotlib.testing.decorators import image_comparison, check_figures_equal
 from matplotlib.transforms import Bbox
 import matplotlib.pyplot as plt
 from matplotlib import (
     collections as mcollections, colors as mcolors, patches as mpatches,
-    path as mpath, style as mstyle, transforms as mtransforms)
+    path as mpath, style as mstyle, transforms as mtransforms, rcParams)
 
 import sys
 on_win = (sys.platform == 'win32')
 
 
+@pytest.fixture()
```

Comment:
-0.5 on this fixture. IMHO the indirection makes it harder to understand while providing little gain.

For the one-liner tests you can equally well use
~~~
assert Patch().get_capstyle() == 'butt'
~~~
for multi-line tests:
~~~
empty_patch = Patch()
~~~
is an extra line, but still easier to follow.